src/components/layout/Categories.tsx

```diff
@@ -2,22 +2,30 @@ import Data from '../../assets/categoriesData.json';
 import CategoryCard from '../core/CategoryCard';
 
 const Categories: React.FC = () => {
-    return (
-        <div className=" m-auto w-full mt-32 mb-10 overflow-x-hidden">
-            <h2 className=" m-auto w-2/3 font-bold text-3xl text-black text-shadow-title"><span className="text-blue-700">Explore</span> By Category</h2>
-            <div className=" flex flex-row w-max flex-wrap gap-5 mt-12">
-                {
-                    Data.map( data =>{
-                        return(
-                            <div className={``} key={data.id}>
-                                <CategoryCard id={data.id} icon={data.icon} title={data.title} description={data.description} colorActive={data.colorActive} colorBasic={data.colorBasic} />
-                            </div>
-                        )
-                    })
-                }
+  return (
+    <div className=" m-auto w-full mt-32 mb-10 overflow-x-hidden">
+      <h2 className=" m-auto w-2/3 font-bold text-3xl text-black text-shadow-title">
+        <span className="text-blue-700">Explore</span> By Category
+      </h2>
+      <div className="flex flex-row max-w-full gap-5 mt-12 overflow-x-scroll">
```

Comment:
Here are the only actual changes, all of them on line 10:

- Add a max-width: 100% to this container via `max-w-full`. This is required so that it fits into its parent all the time.
- Remove `flex-wrap` as it was causing items to go into the next line, when they overflew their container.
- Add the `overflow-x-scroll` class, to make the container scrollable